api/bonsai_api/cli.py

```diff
@@ -109,7 +105,7 @@ def create_user(
 @click.option("-i", "--id", required=True, help="Group id")
 @click.option("-n", "--name", required=True, help="Group name")
 @click.option("-d", "--description", help="Group description")
-def create_group(ctx, id, name, description):  # pylint: disable=unused-argument
+def create_group(_, id, name, description):  # pylint: disable=unused-argument
```

Comment:
Is the `pylint` error still relevant after the rename?

If so, what about turning it off in a pylint config, instead of having to silence it in many places